pho/src/routes/AppRouter.tsx

```diff
@@ -15,6 +16,7 @@ const AppRouter: React.FC = () => {
   return (
     <>
       <Router>
+        <DrawerComponent />
```

Comment:
duplicate with DrawerComponent in App/tsx?

Reply:
removed!